documentation/public/user-guide/introduction.md

```diff
@@ -18,7 +18,7 @@ The Solidity programming language has evolved quite a bit since its inception. S
 
 While it's good for a programming language to evolve and better serve the needs of its users, not being able to easily upgrade or re-deploy existing contracts poses a unique challenge. Developer tooling must be able to understand and consume older contracts that are still being used on the blockchain, written in older versions of Solidity.
 
-Because of that, Slang must be able to reason about different versions of Solidity; how the language grammar, name binding rules, and semantics have changed across different versions. One of our goals is to document differences as part of our [Solidity Specification](../../solidity-specification/supported-versions/).
+Because of that, Slang must be able to reason about different versions of Solidity; how the language grammar, name binding rules, and semantics have changed across different versions. One of our goals is to document differences as part of our [Solidity Specification](../solidity-specification/supported-versions.md).
```

Comment:
nit: shouldn't this link to the top-level page, judging by the link name?

Reply:
split into two links to clarify this. Thanks!